### ISSUE TITLE
On iOS mobile device pod problem

### ISSUE THREAD

**Issue by baykan** [dariocavada/panorama_viewer#6]
Hello, 
It works on android but try to deploy iOS device I have an error.

[!] No podspec found for `dchs_motion_sensors` in `.symlinks/plugins/dchs_motion_sensors/ios`

**Comment by oussabba** [comment]
I have the same issue on IOS

**Comment by dariocavada** [comment]
I'm working on that...

For now there is a workaround:

Use the original plugin in the dependencies section and add the motion sensor override as shown below:

``` yaml 
dependencies:
  panorama: ^0.4.1 

dependency_overrides:
  motion_sensors:
    git:
      url: https://github.com/zesage/motion_sensors
      ref: master

```

**Comment by dariocavada** [comment]
Solved with the version 1.0.2
